geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloBackedRasterOperations.scala

```diff
@@ -290,18 +290,8 @@ object AccumuloBackedRasterOperations {
             shardsConfig: Option[Int],
             writeMemoryConfig: Option[String],
             writeThreadsConfig: Option[Int],
-            queryThreadsConfig: Option[Int],
-            collectStats: Boolean): AccumuloBackedRasterOperations  =
-    if (collectStats)
-      new AccumuloBackedRasterOperations(connector,
-                                         tableName,
-                                         authorizationsProvider,
-                                         visibility,
-                                         shardsConfig,
-                                         writeMemoryConfig,
-                                         writeThreadsConfig,
-                                         queryThreadsConfig) with StatWriter
-    else
+            queryThreadsConfig: Option[Int]): AccumuloBackedRasterOperations  =
```

Comment:
This change should not change anything, the current declaration for AccumuloBackedRasterOperations already uses the StatWriter trait. Unless I am misinterpreting something.